core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -181,7 +188,8 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
         {if (stageData.hasOutput) Seq(("Output Size / Records", "")) else Nil} ++
         {if (stageData.hasShuffleRead) {
           Seq(("Shuffle Read Blocked Time", TaskDetailsClassNames.SHUFFLE_READ_BLOCKED_TIME),
-            ("Shuffle Read Size / Records", ""))
+            ("Shuffle Read Size / Records", ""),
+            ("Shuffle Data Read Remotely", TaskDetailsClassNames.SHUFFLE_READ_REMOTE_SIZE))
```

Comment:
Nit: Maybe a shorter string can be "Shuffle Remote Reads"? It'll make the column thinner.